### ISSUE TITLE
5 states of "oppose" to "support" (e.g. 0.5, -0.5) -- make labels visible for each option

### ISSUE THREAD

**Issue by wrought** [peer/mind#36]
Something more like this, where the labels are visible

![slider with options labeled](https://www.qualtrics.com/wp-content/uploads/2014/01/Slider-Labels.png)

**Comment by mitar** [comment]
https://github.com/Dogfalo/materialize/issues/2751

**Comment by wrought** [comment]
@mitar it seems like this feature we want is enabled in the "click" row of the "Discrete" section as you linked -- so that's not currently supported by the UI library?

**Comment by mitar** [comment]
No. But we could manually style it this way.

**Comment by mitar** [comment]
But those points appear only after you click, to my understanding.

**Comment by mitar** [comment]
Do we even wants steps/discrete slider?

**Comment by mkanwal** [comment]
I definitely think discretization is necessary.  Otherwise, it becomes hard to attach relative meanings to values (as is the case when there are too many discretizations too).

**Comment by mitar** [comment]
Closing in favor of #67 which has more concrete list of improvements to do.

For this ticket, I made it so that the step is 0.25, so you have 8 steps between -1 and 1. I think this is a good compromise between too many steps and not enough steps.
